migrate/20230803154200_scale_timestamp_to_nanos.go

```diff
@@ -0,0 +1,25 @@
+package migrate
```

Comment:
There's no real need to worry about this because all this Postgres code is going to get deleted once state syncer is complete, but I don't think it matters much either way.

Reply:
I added this code here for completeness, so it's one less breaking change in the interim. Nina's and my expectation was that Postgres in its current form will be deprecated long before this new code ships to prod.